packages/flutter/lib/src/cupertino/sliding_segmented_control.dart

```diff
@@ -338,6 +366,17 @@ class CupertinoSlidingSegmentedControl<T extends Object> extends StatefulWidget
   /// The map must have more than one entry.
   final Map<T, Widget> children;
 
+  /// A set to indicate disabled segments.
+  ///
+  /// Disabled children cannot be highlighted and don't show any animation
+  /// when they are tapped or long pressed. So if this set contains [groupValue],
+  /// there is no segment getting highlighted until it is switched to an enabled
+  /// segment.
+  ///
+  /// By default, all segments are selectable. The default text color for
+  /// disabled segments is `Color.fromARGB(115, 176, 176, 176)`.
```

Comment:
I thought maybe it's worth to mention the default text color if the segment is disabled because usually segments show text. Should I remove this part?

Reply:
I'm not sure how helpful this exact value will be in the documentation. It doesn't seem that we want to encourage people to use that exact value in their code (since the value could be updated). Also if the ARGB value of the disabled text color is documented then so should the text weight?

Reply:
I see. Removed this part!